utils/dbutil/threads/pool_test.go

```diff
@@ -16,9 +16,11 @@ func TestMain(m *testing.M) {
 
 func TestThreadPool(t *testing.T) {
 
-	for name, pool := range map[string]ThreadPool{
-		"global": GlobalPool,
-		"local":  ThreadPool{},
+	pool := ThreadPool{}
+
+	for name, pool := range map[string]*ThreadPool{
+		"global": &GlobalPool,
+		"local":  &pool,
```

Comment:
fyi: it is a bit strange, but this also works
```suggestion
		"local":  {},
```